nextflow.config

```diff
@@ -150,7 +150,7 @@ params {
     save_mmseqs_db                       = false
 
     // References
-    genome                     = null
+    //genome                     = null // we use --host_genome instead
```

Comment:
```suggestion
```
or do you want to explicitly document it?

Reply:
Easier for template merges to remember to 'deactivate it'

Reply:
not anymore for future template updates thanks to the `skip_features: [igenomes]`